test/Generator.Tests.Generated/SubStatus.cs

```diff
@@ -8,7 +8,7 @@ namespace Generator.Tests.Generated
     using System.Text.Json.Serialization;
 
     [JsonConverter(typeof(JsonStringEnumConverter))]
-    public enum POISubStatus
+    public enum SubStatus
```

Comment:
It's the right class name, but I think there's a bug lurking here...

Reply:
Umm... Not sure if I was able to spot it. Are you referring to the case ?

Reply:
Because all generated classes and enums are in the same namespace, there will be name collisions when we have two definitions like this:
- `"@id": "dtmi:test:Space:Area:POI:SubStatus;1",`
- `"@id": "dtmi:test:Sensor:SubStatus;1",`

Not something we should address here, but rather open a new issue. (Initial take: we need to begin creating classes and enums in namespaces based on the DTMI provided.)